packages/dx-react-grid/src/plugins/table-column-reordering.jsx

```diff
@@ -63,10 +64,21 @@ export class TableColumnReordering extends React.PureComponent {
   resetCellDimensions() {
     this.cellDimensions = [];
   }
-  storeCellDimensionsGetter(tableColumn, data) {
+  ensureCellDimensionGetters(tableColumns) {
```

Comment:
Are we can put this function in the storeCellDimensionGetter? And configure cellDemenstionGetter in one condition instead of using `delete`.

Reply:
In this case, we can't. We should remove outdated getters from the prev "state".